ICSharpCode.Decompiler/Ast/AstMethodBodyBuilder.cs

```diff
@@ -1133,7 +1133,7 @@ static string FormatByteCodeOperand(object operand)
 			} else if (operand is Cecil.TypeReference) {
 				return ((Cecil.TypeReference)operand).FullName;
 			} else if (operand is VariableDefinition) {
-				return ((VariableDefinition)operand).Name;
+				return ((VariableDefinition)operand).ToString();
```

Comment:
I think this is insufficient: it ought to use the variable name from the PDB... you actually want something like what I did [here](https://github.com/airbreather/ILSpy/commit/4d18cd33e29de480f172b0b23aa4a1fefab845fa#diff-2e552a9f58952a1edca57b6598e8662dR1136) (with the implementation of that over [here](https://github.com/airbreather/ILSpy/commit/4d18cd33e29de480f172b0b23aa4a1fefab845fa#diff-20a9855d63e276cc4b28737af6db6b3cR374))